src/Stryker.Core/Stryker.Core/Initialisation/ProjectFileReader.cs

```diff
@@ -27,6 +28,7 @@ public class ProjectFileReader : IProjectFileReader
         private readonly IBuildalyzerProvider _analyzerProvider;
         private IAnalyzerManager _analyzerManager;
         private readonly ILogger _logger;
+        private readonly StringWriter _buildAlyzerLog = new StringWriter();
```

Comment:
```suggestion
        private readonly StringWriter _buildalyzerLog = new StringWriter();
```
sorry the capital A in the middle is really bothering me :D